cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -158,6 +162,8 @@ def check_for_upgrade_needed():
 
     migrate_from_pre_snaps()
     maybe_install_kube_proxy()
+    update_certificates()
+    create_rbac_resources()
```

Comment:
Can you remove the call to `create_rbac_resources()` here? This should be accounted for by the call to `switch_auth_mode(forced=True)` below, introduced just yesterday by https://github.com/juju-solutions/kubernetes/pull/206.